dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -839,6 +839,10 @@ ERROR_CANT_RESOLVE_FEATURE_PROVIDE_LINK.useraction=Run featureUtility with a com
 
 MSG_USER_FEATURE_SERVER_XML=CWWKF1403I: The following user features are specified in the server.xml and they will not be installed: {0}
 
+ERROR_INCOMPATIBLE_FEATURES_SINGLETON=CWWKF1404E: The singleton features {0} and {1} are incompatible with each other and they cannot be installed at the same time.
```

Comment:
Change
"The singleton features {0} and {1} are incompatible with each other and they cannot be installed at the same time."
To
"**The {0} and {1} singleton features** are incompatible with each other and they cannot be installed at the same time."